daml-foundations/daml-ghc/daml-stdlib-src/DA/Internal/Template.daml

```diff
@@ -51,11 +51,20 @@ archive c = exercise c Archive
 stakeholder : Template c => c -> [Party]
 stakeholder c = signatory c ++ observer c
 
--- | HIDE pass the @c@ using type applications, otherwise it's ambiguous
+-- | Look up the contract id associated with a given contract key.
+--
+-- The `c` needs to be passed using an explicit type application. For
+-- instance, if you want to look up a contract of template `Account` by its
+-- key `k`, you must call `lookupByKey @Account k`.
```

Comment:
Isn't this the one that requires lots of authorization? That feels like an important thing to put in here.